src/service/store/state/state-receipt.ts

```diff
@@ -6,18 +6,27 @@
 import type { Repository } from "@/service/store/repository";
 import { getWeek } from "@/utils/week";
 import { BulletState } from "@/components/bullet/bullet-state";
+import { reactive, readonly } from "vue";
+
 
 export class StateReceipt {
   private readonly repository: Repository;
   private readonly key: string = "receipt";
   private state: Map<string, Date> = new Map<string, Date>();
+  private _status = reactive({
+    bulletState: BulletState.P0,
+  });
 
   constructor(repository: Repository) {
     this.repository = repository;
   }
 
   get = (): Map<string, Date> => this.state;
 
+  get getStatus() {
```

Comment:
this should just be get status()

Reply:
updated, also change the setter to setStatus()